src/components/Matching/styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+import Button from '../Button'
+
+export const Wrapper = styled.section`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+`
+
+export const TitleTabs = styled.div`
+	display: flex;
+	width: 100%;
+
+	& a {
+		flex-grow: 1;
+	}
+`
+
+export const ContentTabs = styled.div`
+	box-sizing: border-box;
+	width: 100%;
+	padding: 1em;
+	border-bottom-left-radius: 0.8em;
+	border-bottom-right-radius: 0.8em;
+	background-color: ${({ theme }) => theme.text.link};
+`
+
+export const StyledButton = styled(Button)<{ active: boolean }>`
+	width: 100%;
+	height: 100%;
+	padding: 1.5em 0;
+	border-bottom-left-radius: 0;
+	border-bottom-right-radius: 0;
+	border: 0.1em solid ${({ theme }) => theme.bg.primary};
```

Comment:
I would prefer if the border was "bolder". :thinking: Like 0.1 is almost invisible, so it seems to me as more of a distraction than intentional feature. How about at least 0.2 or 0.3? But I understand this comes down to a personal preference.

Reply:
I left the border the same as we have for most of the buttons, but I can enlarge it to make it more pronounced 🙂

Reply:
fixed 👍